cmd/ris-mirror/rismirror/vrf.go

```diff
@@ -3,21 +3,23 @@ package rismirror
 import (
 	"github.com/bio-routing/bio-rd/routingtable/locRIB"
 	"github.com/bio-routing/bio-rd/routingtable/mergedlocrib"
+	"github.com/bio-routing/bio-rd/routingtable/vrf"
 )
 
-type _vrf struct {
+type vrfWithMergedLocRIBs struct {
+	vrf         *vrf.VRF
```

Comment:
This field seems to be unused?

Reply:
Not yet. It should be in a following PR by Marvin.